jscomp/bsb/bsb_world.ml

```diff
@@ -64,6 +64,7 @@ let build_bs_deps cwd ~buf (deps : Bsb_package_specs.t) =
 
 let make_world_deps ~cwd ~buf =
   Bsb_log.info "Making the dependency world!@.";
+  Bsb_path_resolver.extract_paths_from_importmap cwd;
```

Comment:
What are you thoughts on making this a config in `bsconfig.json` as well? We could still fallback to `importmap.json`, but I'd personally prefer to have a config property in `bsconfig.json` instead of providing an additional file.

Reply:
> What are you thoughts on making this a config in bsconfig.json as well?

I'm for it :+1: with the caveat that only the imports in the root bsconfig should be read. What do you think?

Reply:
that would be expected, from my perspective.

Reply:
I've made this just accept the parsed bsconfig for now. I think adding an option for a different file is probably premature unless there's feedback from people requesting it.